funnel/templates/loginform.html.jinja2

```diff
@@ -5,6 +5,6 @@
 {% block form %}
   {{ passwordlogin(form, formid, ref_id, action) }}
   {%- if form and form.recaptcha is defined %}
-    {{ recaptcha(ref_id, 'loginformwrapper') }}
+    {{ recaptcha(ref_id, true, 'loginformwrapper') }}
```

Comment:
IMO, the macro should have the `ajax` parameter last, and invocations should explicitly use `ajax=true`. The standalone `true` here is ambiguous, and since we don't have type checking for Jjinja2 templates, this will cause future accidents.